### PR TITLE
7377/7360 - Add CSS utility classes and fix centering of contents in emptymessage

### LINKED ISSUE #7360: EmptyMessage: Is Not Centered on Different Size Widgets
**Describe the bug**
The empty message should be centered when used in a widget.

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/emptymessage/example-widgets.html
2. Add class "card double-height" to one card
3. See that its not centered

**Expected behavior**
Should always center itself.

Possible Solution:
- add a sass mixin for centered (double check if we have one but from my checks we dont)
- sass mixin adds something like
- also add a flex-center utility class
- but make sure empty message auto centers without the class by using the mixn

```
.flex-center {    
 display: flex;
 align-items: center;
 justify-content: center;
 flex-wrap: wrap;
 align-content: center;
 flex-direction: column;
}
```

**Version**
- ids-enterprise 4.82

**Additional context**
[Add any other context about the problem here.
](https://teams.microsoft.com/l/message/19:2b0c9ce520b0481a9ce115f0ca4a326f@thread.skype/1679032110130?tenantId=457d5685-0467-4d05-b23b-8f817adda47c&groupId=4f50ef7d-e88d-4ccb-98ca-65f26e57fe35&parentMessageId=1679032110130&teamName=IDS%20Enterprise%20Development&channelName=General&createdTime=1679032110130&allowXTenantAccess=false)

### LINKED ISSUE #7377: Add CSS utility classes to the project
**Is your feature request related to a problem or use case? Please describe.**
We frequently find ourselves writing repetitive CSS code for commonly used styles such as margins, paddings, and typography. This not only slows down development time but also increases the size of the CSS file.

To improve front-end development efficiency, I suggest adding a set of CSS utility classes to the project. These classes could include commonly used styles such as positioning, padding, margin, font-size, font-weight, text-alignment, and more.

By using these utility classes, developers could save time and reduce the amount of repetitive code they write. Additionally, it would help keep the CSS file size smaller by reducing the number of rules written for specific elements.

**Describe the solution you'd like**
Suggesting that we create a CSS file and add these utility classes including the current ones we have and integrate them into the existing codebase. This would benefit not only the current development team but also future contributors to the project.

Will also need to add this to the doc site as a reference for everyone.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #7440: Display alignment of the empty state is not consistent among widgets
**Describe the bug**
Received a bug ticket in the IDS Jira about empty state images not aligning properly in the widgets. Can we check the padding/alignment across our widgets? Thanks! 

image

Component: Empty State 

Product: LN: Contract Manager

Contact Person: Sudha Addala, Srinithya Pulavarthy 

**Originally submitted as Jira ticket [IDS-968](https://jira.infor.com/browse/IDS-968)**

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR introduces a set of CSS utility classes to the library and enhances the centering of empty messages.

**Related github/jira issue (required)**:

Closes #7377 
Closes #7360 
Closes #7440

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) #7360 Empty Message Fix
- Go to http://localhost:4000/components/emptymessage/test-centered-content.html
- Check all the empty messages
- It should be vertically and horizontally centered on different sizes

2.) #7377 CSS Utilities
- Go to http://localhost:4000/components/css-utilities/example-index.html
- Check and test all CSS utility classes
- Check the documentation readme.md
- Check the _utilities.scss implementation (Open for more suggestions)

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — APPROVED** [review]
Looks good. Maybe we can call it css-utilities to be more specific.
Thats my only suggestion. This came out looking good.

Opening this for QA

**Review by tmcconechy — APPROVED** [review]
@jbrcna @CindyMercadoReyes can you have a quick look

**Comment by jbrcna** [comment]
I already approved this request @tmcconechy . I also noted an observation that .p-left should be 80 instead of 50. Nevertheless, nothing seems to be majorly off.

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@ericangeles i tried to fix the example page but didnt have enough time. can you check http://localhost:4000/components/utilities/example-index.html now that its all 4s the example isnt quite correct.

Also merging main will fix the lint error

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # What's New with Enterprise
 
-## v4.83.0
+## v.4.83.0
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:3]:
```suggestion
## v4.83.0
```

**Comment by ericangeles** [comment]
What's the issue then? I didn't quite understand it.

**Comment by tmcconechy** [comment]
@ericangeles look at the padding example on http://localhost:4000/components/css-utilities/example-index.html
It has a whole bunch of classes that arent in 4.s that done work. i.e. you updated to use 4s but the example doesnt work now.

**Review by jbrcna — APPROVED** [review]

**Review by glenlieorillo — COMMENTED** [review]
QA Passed for [4770](https://github.com/infor-design/enterprise/issues/7440)

image

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**